scripts/operator/raise_pr_to_community_operators_from_our_fork.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import os
+from git import Repo
+from github import Github
+import shutil
+from get_last_published_operator_version import getLastOperatorVersion
+import pr_resources
+
+
+def get_latest_operator_branch():
+    repo = Repo.clone_from(
```

Comment:
isn't there a way to get the list of refs without cloning it? can't you get it from the Github package?

Reply:
I know, that would be great! I've tried with pygit yesterday but didn't actually try with pygithub, I'll have a look!

Reply:
it was so easy :-)